### PR TITLE
added missing activation function in main net

### PR DESCRIPTION
**Pull request by shihui2010**

### CODE REVIEW

File: weight_net.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def __add_layer(self, batch_last_state, out_channels, scope):
             for param in resample_params:
                 print "\tNum of Units: ", param[0], "\tActFun: ", param[1]
             print "Output of the layer:\n\t", batch_rx
-        return batch_rx
+        return tf.nn.sigmoid(batch_rx)
```
Comment (haowu-uiuc) [inline weight_net.py]: This is not correct. Remember, we should not have sigmoid for the output layer (Instead, we use softmax for the output layer, which is written in model.py L31). I have added the activation for main net in my branch, let me push it later.

**Comment by haowu-uiuc** [comment]
abort this PR
